docs/install/install.md

```diff
@@ -138,37 +140,37 @@ Configuration can be done by both environment variable(conf/zeppelin-env.sh) and
     <td>ZEPPELIN_NOTEBOOK_HOMESCREEN</td>
     <td>zeppelin.notebook.homescreen</td>
     <td></td>
-    <td>Id of notebook to be displayed in homescreen ex) 2A94M5J1Z</td>
+    <td>A notebook id displayed in Zeppelin homescreen <br />i.e. 2A94M5J1Z</td>
   </tr>
   <tr>
     <td>ZEPPELIN_NOTEBOOK_HOMESCREEN_HIDE</td>
     <td>zeppelin.notebook.homescreen.hide</td>
     <td>false</td>
-    <td>hide homescreen notebook from list when this value set to "true"</td>
+    <td>This value can be "true" when you want to hide notebooks from Zeppelin homescreen.</td>
   </tr>
   <tr>
     <td>ZEPPELIN_WAR_TEMPDIR</td>
     <td>zeppelin.war.tempdir</td>
     <td>webapps</td>
-    <td>The location of jetty temporary directory.</td>
+    <td>A location of jetty temporary directory</td>
   </tr>
   <tr>
     <td>ZEPPELIN_NOTEBOOK_DIR</td>
     <td>zeppelin.notebook.dir</td>
     <td>notebook</td>
-    <td>Where notebook file is saved</td>
+    <td>A directory path of Zeppelin notebook files</td>
```

Comment:
notebooks are saved in nested subdirectory like `notebook/2A94M5J1Z/note.json`
So maybe this should say "the root directory where notebook directories are saved"